### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Fauzyy**
opslevel-go is a go

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.14
+    working_directory: /go/src/github.com/segmentio/chamber
```
Comment (nickatsegment) [inline .circleci/config.yml]: chamber lives

Comment (Fauzyy) [reply .circleci/config.yml]: haha oops

File: client.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	// ErrServiceUnavailable represents a 422
```
Comment (nickatsegment) [inline client.go:16]: If these map directly to HTTP errors and you're not adding any context, I'd just return the original HTTP error

**Review by nickatsegment — APPROVED** [review]
Nice

File: client.go (hunk 2 above)
Comment (Fauzyy) [reply client.go:16]: That sounds good, I can just return the error body instead of logging it like I do in debugFailedResponse

**Pull request merged**